project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Login/loginForm.html.twig

```diff
@@ -15,9 +15,24 @@
 
         <h1>{{ 'Login'|trans }}</h1>
 
-        {{ form_row(form.email, {label: 'E-mail'|trans} ) }}
 
-        {{ form_row(form.password, {label: 'Password'|trans} ) }}
+        <dl class="form-line">
```

Comment:
Why are these changes necessary

Reply:
Because form attribute `multipleForm: true` changes ID's only for form labels and form rows but not for form_errors so we need remove the form errors so JS validation in login popup works on /login page